chopper_tune.cfg

```diff
@@ -199,14 +204,27 @@ gcode:
             {% set minZAX = minAX %}
             {% set acceleration = settings.printer.max_z_accel|int %}
             {% set trv_speed = config.printer.max_z_velocity|int / 2 * 60 %} ; Idle movements speed
-        {% else %}
-            {action_raise_error('WARNING!!! Incorrect direction')}
+        {% elif axis != 'e'%}
+            {action_raise_error('WARNING!!! Incorrect direction', axis)}
         {% endif %}
 
     {% else %}
         {action_raise_error('WARNING!!! Script does not support your kinematics')}
     {% endif %}
 
+    {% if axis == "e" %}
+        {% set axes = ['e', 'z'] %}
+        {% set steppers = ['extruder'] %}
+        {% set minAX = 0 %}
+        {% set maxAX = 100 %}
+        {% set midAX = maxAX / 2 %}
+        {% set midBAX = 50 %}
+        {% set minZAX = 50 %}
+        {% set acceleration = config.extruder.max_extrude_only_accel|int %}
+        {% set trv_speed = config.extruder.max_extrude_only_velocity|int  / 2 * 60 %}
```

Comment:
This variables for most users are not specified in the configfile, so it's worth using `settings.` instead of `config.`

Reply:
Yes...